src/XMakeTasks/Microsoft.Xaml.targets

```diff
@@ -13,7 +13,7 @@
         the .NET Framework directory and thus will no longer be the right answer. Override it to point
         to the correct .NET Framework location. --> 
    <PropertyGroup>
-      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildFrameworkToolsPath)</XamlBuildTaskPath>
+      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildToolsPath)</XamlBuildTaskPath>
```

Comment:
Sorry! I thought you were replacing the `Microsoft.Xaml.targets` file. When I read this I figured you were importing the one you were replacing. Sorry for the misunderstanding. Looks like you're only dropping the task dll and changing where it looks for it. Got it. This is definitely needed then.